configs/components/rubygem-facter.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-facter' do |pkg, settings, platform|
-  pkg.version '4.0.47'
-  pkg.md5sum '30577ad8e59fb7d6c09dd53115b2a15d'
+  pkg.version '4.0.48'
```

Comment:
Facter 4.0.48 had a bug that was breaking some aggregate facts(FACT-2919).
It was fixed and released in 4.0.49

Reply:
Sweet, updated.